app/routers/sessions.py

```diff
@@ -21,15 +22,20 @@ def str_to_datetime(datetime_str: str) -> datetime:
 
 
 router = APIRouter(prefix="/sessions", tags=["Sessions"])
+logger = get_logger()
 
 
 @router.post("/", response_model=SessionResponse)
 async def create_session(session: Session):
+    logger.info(
```

Comment:
do we have access to session.user_id here before doing `jsonable_encoder(session)`

Reply:
yes